zaza/openstack/charm_tests/neutron_arista/setup.py

```diff
@@ -0,0 +1,39 @@
+# Copyright 2020 Canonical Ltd.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Code for setting up neutron-arista."""
+
+import logging
+import tenacity
+import zaza
+import zaza.openstack.charm_tests.neutron_arista.utils as arista_utils
+
+
+def test_fixture():
+    """Pass arista-virt-test-fixture's IP address to neutron-arista."""
+    fixture_ip_addr = arista_utils.fixture_ip_addr()
+    logging.info(
+        "{}'s IP address is '{}'. Passing it to neutron-arista...".format(
+            arista_utils.FIXTURE_APP_NAME, fixture_ip_addr))
```

Comment:
Could I suggest a slightly different formatting option for consideration?

```python
    logging.info(
        "{}'s IP address is '{}'. Passing it to neutron-arista..."
        .format(arista_utils.FIXTURE_APP_NAME, fixture_ip_addr))
```

It makes the `format` a little easier to read (for me?), but may just be my preference.